space-cli/cmd/utils/filter/filter.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // DeleteOptions filters the resource id based on prefix to delete
-func DeleteOptions(project, prefix string, resources []string, doesResourceExist bool) (string, error) {
+func DeleteOptions(prefix string, resources []string, doesResourceExist bool) (string, error) {
```

Comment:
chnage line number 15 to
		if prefix != "" && strings.HasPrefix(strings.ToLower(resource), strings.ToLower(prefix)) && prefix != resource{

And remove the if at line number 22, always show the filtered resources